### ISSUE TITLE
Ligature Request: ~==

### ISSUE THREAD

**Issue by BenLeggiero** [tonsky/FiraCode#407]
`~=` already becomes `≃`, which is generally used as "bitwise-not assign". I'd like to see `~==` become `≅`, meaning "approximately equal".

**Comment by dhouck** [comment]
Do any existing languages use it that way?

**Comment by BenLeggiero** [comment]
@dhouck I've seen it used (and used it myself) in Swift as a custom operator, again since `~=` looks like it might mean "take this int and reassign it as this other int with all bits flipped", which would be consistent with `&=`, `|=`, `^=`, etc.

**Comment by dhouck** [comment]
Makes sense. I’m also wondering why you prefer `≅` to `≈`? Given that it needs to be three characters wide, I expect ≈ (which I also think is more standard) would look better.

**Comment by BenLeggiero** [comment]
`≈` is fine, too. I was just going by the fact that `~==` has one wavy line and two sets of straight lines, so maybe `≅` would more accurately indicate what the ligature's underlying characters are.

**Comment by dhouck** [comment]
You’re right; it does make more sense with the other characters, especially when you add `===` and `!==` to the mix.
